README.md

```diff
@@ -445,8 +445,59 @@ In this case, the plugin container receives the `--port 8080` arguments. Note th
 You can create encrypted ecdsa and bls keys using the cli which will be needed for operator registration and other onchain calls
 
 ```bash
-eigenlayer operator keys create  --key-type ecdsa [keyname]
-eigenlayer operator keys create  --key-type bls [keyname]
+eigenlayer operator keys create --key-type ecdsa [keyname]
+eigenlayer operator keys create --key-type bls [keyname]
+```
+- `keyname` - This will be name of the created key file. It will be saved as `<keyname>.ecdsa.key.json` or `<keyname>.bls.key.json`
```

Comment:
```suggestion
- `keyname` - This will be the name of the created key file. It will be saved as `<keyname>.ecdsa.key.json` or `<keyname>.bls.key.json`
```